### PR TITLE
FeatureGate documentation is incomplete and incomprehensible

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1860289

### CODE REVIEW

File: modules/nodes-cluster-enabling-features-cluster.adoc [hunk 1]
```diff
@@ -40,9 +29,29 @@ kind: FeatureGate
...
+  enabled:
+    - VolumePVCDataSource
+    - VolumeSnapshotDataSource
+    - BlockVolume
+    - CSIBlockVolume
 ----
-<1> Add the `featureSet: "TechPreviewNoUpgrade"` parameter.
+<1> Add the `featureSet: parameter and the feature gates you want to activate.
+<2> For the `CustomNoUpgrade` feature set, specify the features you want to 
+enable or disable.
```
Comment (mburke5678) [inline modules/nodes-cluster-enabling-features-cluster.adoc:45]: @sttts Can the user get lists of features that the TechPreviewNoUpgrade and CustomNoUpgrade affect? 

We have this unused topic that lists features that can be turned on/off. But this seems unmanageable from version to version

Comment (sttts) [reply modules/nodes-cluster-enabling-features-cluster.adoc:45]: I don't think users can get a list.

`oc explain ...` could output such a list if the API docs have it. Today this is not the case.

cc @danwinship @deads2k @rphillips, who conceived this API afaik.

Comment (danwinship) [reply modules/nodes-cluster-enabling-features-cluster.adoc:45]: I wasn't involved with the design of the API. I just added the `IPv6DualStackNoUpgrade` feature set.

FTR as far as I could tell, the "prevents upgrade" stuff in the API docs is a lie. I couldn't find anything anywhere that actually prevents upgrades if you are using a "NoUpgrade" featureset

**Comment by sttts** [comment]
Is this resolved elsewhere?

**Comment by mburke5678** [comment]
@sttts https://github.com/openshift/openshift-docs/pull/24132

**Pull request closed without merging**